packages/button/theme.module.css

```diff
@@ -81,8 +81,16 @@
     --scrim-color: var(--brand);
   }
 
-  &.background-dark,
   &.background-brand {
+    --border-color-hovered: transparent;
+    --border-color: rgba(255, 255, 255, 0.24);
+    --scrim-color: var(--white);
+    --scrim-opacity-active: 0.08;
+    --scrim-opacity-hovered: 0.14;
```

Comment:
This is slightly duplicative, but I figured it might be best to be explicit. This change has also got me thinking about `button` needing some theme intention love... There are so many combinations of `{ brand, variant, background }` that result in identical combinations, that I wonder whether it might be preferable to rethink how we approach themeing. Added a comment to [this task](https://app.asana.com/0/1100423001970639/1200880473915564/f) since it feels like it's in a similar vein of "restructure Button".

> Additional context: Splitting out `<a />` vs `<button />` buttons may be related, as generally, the places where we do super custom coloration for button are places where it's really an anchor tag. Anchor tags have fewer theme-related style concerns, e.g. no `disabled` state, which means the control we need for custom theming may be simpler to implement with a more scoped "link button" component.